fem/bilinearform.cpp

```diff
@@ -970,6 +970,9 @@ void BilinearForm::EliminateVDofs(const Array<int> &vdofs_,
                                   DiagonalPolicy dpolicy)
 {
    vdofs_.HostRead();
+   mat->HostReadWriteI();
+   mat->HostReadWriteJ();
+   mat->HostReadWriteData();
```

Comment:
Shouldn't these go inside `SparseMatrix::EliminateRowCol`?

Reply:
done in [caa79bf](https://github.com/mfem/mfem/pull/3127/commits/caa79bf80147c2b36b3f1070ec890221cbe2f11d)